src/ptests/para_high_order.c

```diff
@@ -0,0 +1,334 @@
+/*
+! @file para_high_order.c
+! @author Laurent Cadet <laurent.cadet@cenaero.fr>
+! @version 0.1
+!
+! @section LICENSE
+! BSD style license
```

Comment:
CGNS software should use CGNS license (zlib/libpng modified license)

Reply:
This testcase is a copy of the src/ptests/test_unstructured.c. Thus its header is the same ...
should I correct all of them ?